common/Resources/ti.kernel.js

```diff
@@ -1,18 +1,36 @@
+// This is the file each platform loads on boot *before* we launch ti.main.js to insert all our shims/extensions
+// and eventually load app.js
+// This is where any common setup cross-platform should take place
+// This is analagous to android's runtime/common/src/js directory before
+// On Android, this will get baked into the binary as raw char* to be loaded
+// On iOS, we'll simply evaluate/execute this before launching ti.main.js
+
+// So what does Android do here?
+// extends it's baked in EventEmitter
+// Sets up require via module.js
+// Loads path.js so it can do module.js
+// Does special lazy hooking/bootstrapping of Titanium APIs/Proxy
+
 /**
  * Appcelerator Titanium Mobile
  * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
  * Licensed under the terms of the Apache Public License
  * Please see the LICENSE included with this distribution for details.
  */
-'use strict';
+/* globals OS_IOS,OS_ANDROID */
```

Comment:
<!--
  0 failure: 
  1 warning:  common/Resources/...
  
  
  DangerID: danger-id-Danger;
  File: common/Resources/ti.kernel.js;
  Line: 20;
-->

- :warning: common/Resources/ti.kernel.js line 20 – 'OS_IOS' is defined but never used. (no-unused-vars)